manta-js/package/src/api/index.js

```diff
@@ -117,26 +121,42 @@ export default class Api {
       await this.api.isReady;
 
       this._log('checkpoint ' + JSON.stringify(checkpoint));
-      let result = await this.api.rpc.mantaPay.pull_ledger_diff(
+      let result = await this.api.rpc.mantaPay.dense_pull_ledger_diff(
```

Comment:
Can we remove `pull_ledger_diff` rpc call?

If its not going to being used in sdk why do we need it?

Reply:
We probably should keep it around for at least the near term

Reply:
I removed related types for `pull_ledger_diff`.
https://github.com/Manta-Network/sdk/pull/71/files#diff-a4cdaf8fc2c394d9fa1a99067d2cb0a6b55d844167de127b24e2bdb8c8c633acL86

Reply:
So why keep it in the runtime altogether ?

Reply:
> So why keep it in the runtime altogether ?

So that a rollback of this change can be done without needing a runtime
upgrade